proto/tsopb.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+package tsopb;
+
+import "pdpb.proto";
+
+import "gogoproto/gogo.proto";
+import "rustproto.proto";
+
+option (gogoproto.sizer_all) = true;
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (rustproto.lite_runtime_all) = true;
+
+option java_package = "org.tikv.kvproto";
+
+service TSO {
+    rpc Tso(stream pdpb.TsoRequest) returns (stream pdpb.TsoResponse) {}
+
+    rpc SyncMaxTS(pdpb.SyncMaxTSRequest) returns (pdpb.SyncMaxTSResponse) {}
+
+    rpc GetDCLocationInfo(pdpb.GetDCLocationInfoRequest) returns (pdpb.GetDCLocationInfoResponse) {}
+
+    rpc SetExternalTimestamp(pdpb.SetExternalTimestampRequest) returns (pdpb.SetExternalTimestampResponse) {}
+
+    rpc GetExternalTimestamp(pdpb.GetExternalTimestampRequest) returns (pdpb.GetExternalTimestampResponse) {}
+}
```

Comment:
need new line？

Reply:
I saw the services definition in the other .proto files  have the same style - a new line between two rpcs, so I use the same style.

Reply:
BTW, I'll hold this PR until I completed the gPRC server/client change in tikv/pd repo.

Reply:
just realized that you meant the new line at the end of file. Added.